python/interpret_community/widget/explanation_dashboard.py

```diff
@@ -168,25 +159,58 @@ def __init__(self, explanation, model=None, *, dataset=None,
                 raise e
         ExplanationDashboard.service.use_cdn = use_cdn
         ExplanationDashboard.model_count += 1
-        predict_url = "http://{0}:{1}/{2}/predict".format(
-            ExplanationDashboard.service.ip,
-            ExplanationDashboard.service.port,
-            str(ExplanationDashboard.model_count))
-        ExplanationDashboard.explanations[str(ExplanationDashboard.model_count)] =\
+        if (not in_cloud_env):
+            predict_url = "http://{0}:{1}/{2}/predict".format(
```

Comment:
have you tried the notebook vm trick to make this work there?

Reply:
nvm, read the intro to PR, it looks like that's not part of this PR

Reply:
Yeah, I want that to work, but it looks like it might not be trivial....